blazar/db/sqlalchemy/api.py

```diff
@@ -1485,10 +1522,18 @@ def network_extra_capability_get_all_per_network(network_id):
 
 def network_extra_capability_create(values):
     values = values.copy()
+
+    resource_property = _resource_property_get_or_create(
```

Comment:
What if there is no capability_name? What happens if `None` is passed in here. I don't see any protection against that.

Reply:
The capability_name is a none nullable field in the ExtraCapability model so it should throw an error is none is passed. Might be a nest unit test.